src/status_im2/subs/contact_test.cljs

```diff
@@ -0,0 +1,183 @@
+                                        :contact-request-state 1}
+                             "0xtest2" {:last-updated          1672582629695
+                                        :mutual?               true
+                                        :contactRequestClock   0
+                                        :images                {}
+                                        :added                 true
+                                        :name                  "slim.shady"
+                                        :Removed               false
+                                        :trustStatus           0
+                                        :alias                 "Fake Slim Shady"
+                                        :bio                   ""
+                                        :IsSyncing             false
+                                        :display-name          ""
+                                        :ens-verified          true
+                                        :socialLinks           nil
+                                        :blocked               false
+                                        :verificationStatus    0
+                                        :lastUpdatedLocally    1672582563204
+                                        :public-key            "0xtest"
+                                        :has-added-us          true
+                                        :contact-request-state 1}
+                             "0xtest3" {:last-updated          1672582629695
+                                        :mutual?               true
+                                        :contactRequestClock   0
+                                        :images                {}
+                                        :added                 true
+                                        :name                  "slim.shady"
+                                        :Removed               false
+                                        :trustStatus           0
+                                        :alias                 "Instant noodles"
+                                        :bio                   ""
+                                        :IsSyncing             false
+                                        :display-name          ""
+                                        :ens-verified          true
+                                        :socialLinks           nil
+                                        :blocked               false
+                                        :verificationStatus    0
+                                        :lastUpdatedLocally    1672582563204
+                                        :public-key            "0xtest"
+                                        :has-added-us          true
+                                        :contact-request-state 1}}})
+
+(def ^:private expected-sorted-contacts
+  '({:title "F"
```

Comment:
No need to use lists in test assertions. Just replace them with vectors. This is more idiomatic in Clojure tests because equality of `(is (= [] '()))` is true. By using vectors everywhere in tests there's no need to *quote* collections with the `'` character.

It's rare to see tests that actually care about strict equality. In general, it doesn't matter for the test if the return value is a lazy collection returned by a `map` call or if the return value is a vector returned by `mapv`.

```clojure
[{:title "F"
  :data [{:active? true]
          :last-updated 1672582629695
          ...
  ...
```